ui/src/App.test.js

```diff
@@ -1,8 +1,110 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { fireEvent, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import App from "./App";
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe("App", () => {
+  beforeAll(() => {
+    jest.useFakeTimers("modern");
+    const date = new Date("2022-02-15T04:00");
+    jest.setSystemTime(date);
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  it("renders correctly", () => {
+    const app = render(<App />);
+    expect(app.asFragment()).toMatchSnapshot();
+  });
+
+  it("shows calander", async () => {
+    render(<App />);
+    expect(await screen.findByText("Month")).toBeVisible();
```

Comment:
Are there any other expectations we can add to prove the calendar has rendered? This feels a bit sparse

Reply:
We can add that we see the days of the week show up (Monday, Tuesday etc..)

Reply:
I ended up discovering that `react-big-calendar` has an aria label attached to each of it's calendar view, and for month view the label is "Month View" so I checked that this element was visible via it's aria label. That will ensure the entire table view is showing up unless the package is self end up having some kind of bug introduced to it (but not likely)

Reply:
![Screen Shot 2022-10-23 at 3 30 23 PM](https://user-images.githubusercontent.com/50315144/197421429-8632f0e0-54ff-4868-a9eb-d868a5cb74a5.png)